src/kubernetes.h

```diff
@@ -122,6 +140,15 @@ class KubernetesReader {
       const json::Object* pod, Timestamp collected_at, bool is_deleted) const
       throw(json::Exception);
 
+  // Node Resource Mapping Callback.
+  IdsAndMR NodeResourceMappingCallback(const json::Object* node)
+      const throw(json::Exception);
```

Comment:
You can fit `const` on the previous line. Also below.

Reply:
done